src/App.jsx

```diff
@@ -30,6 +31,10 @@ const App = () => {
           <Route path='/login' element={<LoginPage />} />
           <Route path='/signup' element={<SingupPage />} />
           <Route path='/product/:id' element={<SingleProductPage />} />
+          <Route
+            path='/dashboard/users'
+            element={<AdminManageUserContainer />}
```

Comment:
Please only pages should appear here, rename or create another file. Follow the conventions we agreed on.

Reply:
ok